packages/styles/scss/patterns/simplelongform/_simplelongform.scss

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin simplelongform { 
+  .#{$prefix}--simplelongform {
+    color: $text-01;
+    background: $ui-background;
+    &__container {
+      padding-left: $layout-01;
+      padding-right: $layout-01;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__col {
+      @include carbon--make-col-ready;
+      @include carbon--make-col-offset(4, 16);
+      @include carbon--make-col(7, 16);
+    }
+    &__title {
+      @include type-style('expressive-heading-05');
+    }
+    &__content {
+      @include type-style('body-long-02');
+    }
+    &__divider {
+      border-bottom: 1px solid $ui-04;
+      &__col {
+        @include carbon--make-col-offset(4, 16);
+        @include carbon--make-col(7, 16);
+      }
+    }
+    &__link {
+      @include type-style('body-short-02');
+      &__col {
+        margin-top: $layout-03;
+      }
+      @include carbon--breakpoint('sm') { 
+        &__col {
+          @include carbon--make-col-ready;
+          @include carbon--make-col-offset(0, 4);
```

Comment:
You shouldn't need offset at the smallest breakpoint for this component.

Reply:
right